hypha/static_src/javascript/esm/choices.js-10-2-0.js

```diff
@@ -4,5 +4,5 @@
ow new Error(c(6));t=!1,p();var i=a.indexOf(e);a.splice(i,1),o=null}}}function v(e){if(!function(e){if("object"!=typeof e||null===e)return!1;for(var t=e;null!==Object.getPrototypeOf(t);)t=Object.getPrototypeOf(t);return Object.getPrototypeOf(e)===t}(e))throw new Error(c(7));if(void 0===e.type)throw new Error(c(8));if(h)throw new Error(c(9));try{h=!0,s=r(s,e)}finally{h=!1}for(var t=o=a,i=0;i<t.length;i++)(0,t[i])();return e}return v({type:u.INIT}),(n={dispatch:v,subscribe:m,getState:f,replaceReducer:function(e){if("function"!=typeof e)throw new Error(c(10));r=e,v({type:u.REPLACE})}})[l]=function(){var e,t=m;return(e={subscribe:function(e){if("object"!=typeof e||null===e)throw new Error(c(11));function i(){e.next&&e.next(f())}return i(),{unsubscribe:t(i)}}})[l]=function(){return this},e},n}var p=d;function f(e){for(var t=Object.keys(e),i={},n=0;n<t.length;n++){var r=t[n];"function"==typeof e[r]&&(i[r]=e[r])}var s,o=Object.keys(i);try{!function(e){Object.keys(e).forEach((function(t){var i=e[t];if(void 0===i(void 0,{type:u.INIT}))throw new Error(c(12));if(void 0===i(void 0,{type:u.PROBE_UNKNOWN_ACTION()}))throw new Error(c(13))}))}(i)}catch(e){s=e}return function(e,t){if(void 0===e&&(e={}),s)throw s;for(var n=!1,r={},a=0;a<o.length;a++){var l=o[a],h=i[l],u=e[l],d=h(u,t);if(void 0===d)throw t&&t.type,new Error(c(14));r[l]=d,n=n||d!==u}return(n=n||o.length!==Object.keys(e).length)?r:e}}function m(e,t){return function(){return t(e.apply(this,arguments))}}function v(e,t){if("function"==typeof e)return m(e,t);if("object"!=typeof e||null===e)throw new Error(c(16));var i={};for(var n in e){var r=e[n];"function"==typeof r&&(i[n]=m(r,t))}return i}function g(){for(var e=arguments.length,t=new Array(e),i=0;i<e;i++)t[i]=arguments[i];return 0===t.length?function(e){return e}:1===t.length?t[0]:t.reduce((function(e,t){return function(){return e(t.apply(void 0,arguments))}}))}function _(){for(var e=arguments.length,t=new Array(e),i=0;i<e;i++)t[i]=arguments[i];return function(e){return function(){var i=e.apply(void 0,arguments),n=function(){throw new Error(c(15))},r={getState:i.getState,dispatch:function(){return n.apply(void 0,arguments)}},s=t.map((function(e){return e(r)}));return n=g.apply(void 0,s)(i.dispatch),a(a({},i),{},{dispatch:n})}}}}},t={};function i(n){var r=t[n];if(void 0!==r)return r.exports;var s=t[n]={exports:{}};return e[n].call(s.exports,s,s.exports,i),s.exports}i.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return i.d(t,{a:t}),t},i.d=function(e,t){for(var n in t)i.o(t,n)&&!i.o(e,n)&&Object.defineProperty(e,n,{enumerable:!0,get:t[n]})},i.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},i.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})};var n,r,s={};return n=i(373),r=i.n(n),i(187),i(883),i(789),i(686),s.default=r(),s=s.default}()};var i=t.exports=e();export{i as default};
```

Comment:
@theskumar  This file should be reverted I think, looks like a mistake in a search and replace.

Get an error from babel when trying to build.